Signal/src/ViewControllers/ConversationView/Cells/OWSMessageBubbleView.m

```diff
@@ -920,15 +918,14 @@ - (CGSize)quotedMessageSizeForViewWidth:(int)viewWidth
     return result;
 }
 
-- (CGSize)sizeForViewWidth:(int)viewWidth contentWidth:(int)contentWidth
+- (CGSize)sizeForContentWidth:(int)contentWidth
```

Comment:
Currently, the message bubbles look exactly the same in conversation view and in message detail view. For example, they only occupy 80% of the "content width".  Incoming messages hug leading edge, outgoing messages hug trailing edge. 

We could change that so they are more "greedy".

They also skip their "tail" if they would in conversation view.  We could change that too.  I think I will.

Reply:
> They also skip their "tail" if they would in conversation view. We could change that too. I think I will.

I just made this change.  It's not in the screenshots.